src/main/java/uk/gov/cslearning/catalogue/service/EventService.java

```diff
@@ -88,18 +88,20 @@ public Event save(String courseId, String moduleId, Event event){
     public Event getEventAvailability(Event event) {
         List<Booking> bookings = learnerRecordService.getEventBookings(event.getId());
 
-        event.getVenue().setAvailability(event.getVenue().getCapacity());
-
-        bookings.forEach(b -> {
-            if (b.getStatus() == BookingStatus.CONFIRMED || b.getStatus() == BookingStatus.REQUESTED) {
-                event.getVenue().setAvailability(event.getVenue().getAvailability() - 1);
-            }
-        });
+        if(event.getVenue() != null) {
```

Comment:
don't bundle in your changes together. Makes life easier to keep the delta change to a minimum. When things break you know what caused it. Also if we have to prioritize one bug over another, now they are bundled and we can't.